tests/test_cmdline.py

```diff
@@ -600,7 +600,7 @@ def test_command_overrides(monkeypatch, capsys, console):
     # Invoke a command that is known to have its own custom arguments
     # (In this case, the channel argument for publication)
     cmd, options, overrides = do_cmdline_parse(
-        "publish macos app -C version='1.2.3' -C extra=42".split(),
+        ["publish", "macos", "app", "-C", "version='1.2.3'", "-C", "extra=42"],
```

Comment:
> ruff fixed SIM905 and I think it's better, although not a huge different either way.

It would be even better if `do_cmdline_parse` accepted either a list or a string, and automatically converted the string using `shlex.split`.

Reply:
Given it's a test utility method, and one that is only used in this test file, I'm inclined to think only accepting the list form is more appropriate - it's more explicit, and the extra characters to split the parse call into extra arguments isn't especially onerous.